test/e2e/utils/command.go

```diff
@@ -24,10 +24,21 @@ import (
 	"github.com/medik8s/fence-agents-remediation/api/v1alpha1"
 )
 
+// StopKubelet runs cmd connand to stop kubelet for the node and returns an error only if it fails
+func StopKubelet(c *kubernetes.Clientset, nodeName string, testNsName string, log logr.Logger) error {
+	cmd := "microdnf install util-linux -y && /usr/bin/nsenter -m/proc/1/ns/mnt /bin/systemctl stop kubelet"
+	_, err := runCommandInCluster(c, nodeName, testNsName, cmd, log)
+	if err != nil && strings.Contains(err.Error(), "connection refused") {
+		log.Info("ignoring expected error when stopping kubelet", "error", err.Error())
+		return nil
+	}
+	return err
+}
+
 // GetBootTime gets the boot time of the given node by running a pod on it executing uptime command
```

Comment:
nit: function description should not state how the function works, just what it is expected to do (also, not part of this PR, so no need to change it here)